### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/keptn-go-utils/commit/0751cd6f9b4619cd29b1ac44775192a755e8a958#commitcomment-46997367

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/keptn-go-utils/commit/33b8289bddb131df0eb13343192011dd975ccf58

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: pkg/lib/keptn/keptn_base.go [hunk 1]
```diff
@@ -216,10 +216,10 @@ func ValidateKeptnEntityName(name string) bool {
 	return len(processedString) == len(name)
 }
 
-// ValididateUnixDirectoryName checks whether the provided dirName contains
+// ValidateUnixDirectoryName checks whether the provided dirName contains
 // any special character according to
 // https://www.cyberciti.biz/faq/linuxunix-rules-for-naming-file-and-directory-names/
-func ValididateUnixDirectoryName(dirName string) bool {
+func ValidateUnixDirectoryName(dirName string) bool {
```
Comment (jsoref) [inline pkg/lib/keptn/keptn_base.go:222]: This is an API break -- https://github.com/keptn/keptn/pull/3234#discussion_r573863446

It probably makes sense to split this off into its own PR. But, I'm leaving it here until someone provides feedback.

Comment (johannes-b) [reply pkg/lib/keptn/keptn_base.go:222]: Hi @jsoref, I agree with you! Please move this change into another PR. 

Other than that, this PR looks good. Thanks!

**Review by johannes-b — APPROVED** [review]
lgtm
